src/test/java/com/qa/automation/runners/TestRunner.java

```diff
@@ -9,10 +9,12 @@
 @CucumberOptions(
         features="classpath:features",//to tell cucumber where is ur feature file
         glue="com.qa.automation.stefdefs", // to tell cucumber where is ur step def code
-        tags="@LogoTitleValidation", // to tell which tagged feature file to execute
```

Comment:
pleasee remove the tags. the "tags" should be empty.

Reply:
made the changes please check and approve the  PR

Reply:
It Seems like the changes are good for now. approved.